From eb871588e75c00b7b4818047367fe85a3007040b Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Wed, 18 May 2011 18:42:22 -0400
Subject: [PATCH 144/248] lttng-relay-refactoring

LTTng relay refactoring

- Change design to inheritance for channels and buffers.
  - Eliminate structure duplication for locking/allocation level of buffer and
    channel.
  - Remove now unnecessary krefs.
- Fix cpu hotplug for timers.
- Cleanups

Impact:
- Cleaner code
- Smaller d-cache footprint
- Complete license change for LTTng parts re-usable for userspace tracing to
  dual-license LGPL v2.1 / GPL v2.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 include/linux/marker.h |    1 -
 kernel/marker.c        |    2 +-
 2 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/include/linux/marker.h b/include/linux/marker.h
index debfe9a..c92a369 100644
--- a/include/linux/marker.h
+++ b/include/linux/marker.h
@@ -15,7 +15,6 @@
 #include <stdarg.h>
 #include <linux/types.h>
 #include <linux/immediate.h>
-#include <linux/ltt-channels.h>
 
 struct module;
 struct marker;
diff --git a/kernel/marker.c b/kernel/marker.c
index 9885cea..86820a1 100644
--- a/kernel/marker.c
+++ b/kernel/marker.c
@@ -1267,7 +1267,7 @@ __initcall(init_markers);
 
 #endif /* CONFIG_MODULES */
 
-void ltt_dump_marker_state(struct ltt_trace_struct *trace)
+void ltt_dump_marker_state(struct ltt_trace *trace)
 {
 	struct marker_entry *entry;
 	struct ltt_probe_private_data call_data;
-- 
1.7.0.4

